050-AIS-APIManagementwithFunctions/Student/Challenge-04.md

```diff
@@ -0,0 +1,73 @@
+# Challenge 04 - Securing Backends APIs
+
+
+[<Previous Challenge](./Challenge-03.md) - **[Home](../README.md)**
+
+## Pre-requisites
+
+- You should have completed Challenge 03
+
+## Introduction
+You would like to be able to secure your backend APIs in one of the two ways:
+- Secure Hello API in a private network
+- Test end-to-end authorization to Hello API via OAuth.  
+
+## Description
+You should be able to either:
+- For Scenario 01: Configure secured backend APIs in a private network
```

Comment:
Make this a sub-header with 3 hashes.

"### Scenario 1: Configure secured backend APIs in a private network"

Reply:
Done